data/khalidx3.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Pr for Adding your profile already open so you can remove it from here 

I think you actually made changes in the same fork as profile addition that's why there are same changes in both of your pr

Reply:
Alright got it that's my bad will do it.